plugins/badges/src/components/EntityBadgesDialog.tsx

```diff
@@ -87,6 +71,10 @@ export const EntityBadgesDialog = ({ open, onClose, entity }: Props) => {
       <DialogTitle>Entity Badges</DialogTitle>
 
       <DialogContent>
+        <Typography>
```

Comment:
`DialogContentText`.

Reply:
Will do.  It's interesting, as there's actually a look in here so it ends up being...

* DialogContent
  * Typograhy
  * DialogContentText (each badge)
  * DialogContentText (each badge)
  * DialogContentText (each badge)

But updated that first one per your feedback and tested locally.